ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -197,10 +197,16 @@ class TFLiteInterpreter
   /** @brief cache input and output tensor ptr before invoke */
   int cacheInOutTensorPtr ();
 
-  /** @brief set delegate for the tflite interpreter */
-  void setDelegate (TfLiteDelegate *delegate)
+  /** @brief get delegate for the tflite interpreter */
```

Comment:
May be a minor typo here.

Reply:
Thank you for the comment - typo fixed in updated patchset